site/docs/guides/audio.md

```diff
@@ -196,16 +196,25 @@ w4.tone(262 | (523 << 16), 60, 100, w4.TONE_PULSE1);
 </MultiLanguageCode>
 
 ## ADSR Envelope
+ 
+### Volume 
+
+There are two volume levels That can be passed to the tone function
+
+| Volume  | Shift | Description                                            |
+| ------- | ----- | ------------------------------------------------------ |
+| Sustain | 0     | The volume sustained for the `sustain` phase duration. |
+| Peak    | 16    | The volume reached by the `attack` phase               |
 
 [ADSR](https://en.wikipedia.org/wiki/ADSR_envelope) describes how the volume changes over time, and
 has 4 time components:
 
-| Time        | Shift | Description                                                                  |
+| Time        | Shift | Description                                                                    |
 | ---         | ---   | ---                                                                          |
 | **A**ttack  | 24    | The time it takes to initially ramp up from 0 volume to 100% volume.         |
```

Comment:
Should say "peak volume" here, not 100%

Reply:
I thought I changed that 🤷

Reply:
Ahh I accidentally discarded it when removing some formatting changes.